style-guide/_colors.scss

```diff
@@ -2,6 +2,7 @@ $palette_white:              #fff;    // Safe to use with $palette_grey_text –
 // Shades of gray from lighter to darker.
 $palette_grey_ultra_light:   #f7f7f7; // Safe to use with $palette_grey_text – $palette_black.
 $palette_grey_light:         #f1f1f1; // Safe to use with $palette_grey_text – $palette_black.
+$palette_grey_medium_light:  #e2e4e7;
```

Comment:
Did you discuss adding this color to the style guide with design? If not, please first make sure they are ok with it, otherwise use one of the existing colors from the style guide.

Reply:
This has been discussed with @hedgefield and he was fine with it 😄 But good question to ask!